configuration/values.yaml

```diff
@@ -0,0 +1,106 @@
+global:
+  dockerRegistry:
+    url: ghcr.io/nationalsecurityagency
+  volumes:
+    certificates:
+      name: certificates
+      destination: /data/certs
+    microserviceConfig:
+      name: microservice-config
+      destination: /microservice-config
+  initialization:
+    image:
+      name: busybox
+      tag: 1.28
+  certSecretName: certificates-secret
+  certificates:
+    externalSecret:
+      enabled: false
+      name: ""
+  keystorePassword: changeme
+  truststorePassword: changeme
+  externalConfigMap:
+    enabled: false
+    name: ""
+  secrets:
+    keystore:
+      alias: "1"
+      password: changeme
+      path: "file:///data/certs/keystore.p12"
+      type: PKCS12
+    truststore:
+      password: changeme
+      path: file:///data/certs/truststore.jks
+      type: JKS
+  accumulo:
+    instanceName: dev
+    password: ThisP@ssw0rd1sBANANAS
```

Comment:
move to a secret?

Reply:
Yes, but not in this ticket. We can work that after.